doc/api/http.md

```diff
@@ -1210,7 +1212,8 @@ added: v0.4.0
 
 Sets a single header value for implicit headers. If this header already exists
 in the to-be-sent headers, its value will be replaced. Use an array of strings
-here to send multiple headers with the same name.
+here to send multiple headers with the same name. Values are coerced to
+strings if needed during transmission.
```

Comment:
Perhaps just this for the last sentence? `Non-string values will be coerced to strings.`